### DISCUSSION TITLE
Windows Logon-Screen

### DISCUSSION THREAD

**Discussion by stephannn** [rustdesk/rustdesk#9056] [Q&A]
Hi there,
maybe I just don't find the setting. We are currently testing RustDesk Pro and have our agents to be configured to accept an incoming connection by one-click, no password, because only admin users can connect anyways.

Now when I restart the machine, I see the client is online (RustDesk runs as a service), I can send a request but:
  1st: the end-user cannot accept it
  2nd: it would be nice if we can connect without any confirmation because there is no active session on the client.

In special, does the second point even exists?

**Comment by rustdesk** [comment]
Yes, you have answered yourself. This is impossible for logon-screen with on-click enabled only.

**Reply by stephannn** [reply]
Hi,
but how do I know the password if I cannot see the client app?
Besides, I already have groups and users configured who can connect. The SCCM Remote tool offers following settings for this kind of problem:
![image](https://github.com/user-attachments/assets/c95be6eb-46b8-4fa8-9ef8-1550fcc6406d)
So I restart a device, and afterwards, as long as no user is connected, my admin user is elevated to connect

Edit:
I checked it again and you are right, if I enable password authentication I get the pop-up to enter the password. 

However, I don't think this "workaround" is very enterprise friendly. Sometime we connect to a machine, and perform a restart where we must connect again. It is not really secure to use just one password for all the clients, at the same time, we won't note the password all the time that a specific client had.
In my opinion, when you set up users and groups, there should be the option that these users that are already managed, are able to connect to a not used device, even when the use another OIDC. Please keep in mind that I am talking about the Pro Version.
